addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -124,30 +124,86 @@ const PATH_END_REASONS = {
  * Creates a generator function according to the given parameters. Pre-made
  * generators to traverse the DOM are made using this function:
  *
- * @see leftDeepFirstPath
- * @see leftDeepOnlyPath
- * @see leftDeepFirstInlinePath
- * @see leftDeepOnlyInlinePath
- *
- * @see rightDeepFirstPath
- * @see rightDeepOnlyPath
- * @see rightDeepFirstInlinePath
- * @see rightDeepOnlyInlinePath
+ * @see leftLeafFirstPath
+ * @see leftLeafOnlyNotBlockPath
+ * @see leftLeafOnlyInScopeNotBlockPath
```

Comment:
![image](https://user-images.githubusercontent.com/24205914/126990014-55798020-229f-4557-a32e-ec224cb6b51c.png)

Reply:
Especially in a "REF" commit.

Reply:
Being a refactor does not mean we need more function.
We actually have the method only for the case encountered in the code.

Reply:
> Being a refactor does not mean we need more function.

It doesn't but it raises questions when in a refactor you get rid of a function that existed before. I take it it was just not in use...